acp/acp_local.go

```diff
@@ -105,6 +107,40 @@ func (l *ACPLocal) Close() error {
 	return l.manager.Terminate()
 }
 
+func (l *ACPLocal) DropAll(ctx context.Context) error {
+	err := l.Close()
```

Comment:
question: `l.manager.Terminate()` is safe to call close on even if it was already closed before?

Reply:
Yea there might be an issue here, thanks for flagging

Reply:
Resolved this, can you do another quick review on the last 2 commits. Added a flag to track if we're already closed so we don't duplicate the termination stuff. @shahzadlone

Reply:
Re-reviewed. Just left one suggestion